### ISSUE TITLE
Standardization of terms

### ISSUE THREAD

**Issue by saltzberg** [salilab/pmi#163]
We should standardize nomenclature in PMI.  For example, in our current working code:

Should the internal model object be self.m or self.mdl?
Is the crosslink database object passed to XL restraints passed as CrossLinkDataBase or xldb?
What is the base object passed to restraints?  PMI Objects or Hierarchies?

**Comment by cgreenberg** [comment]
Agreed :)
Perhaps we should write a "developer guide" for ourselves?

**Comment by saltzberg** [comment]
Yes, I think some simple guidelines would be a good idea.  Let's post thoughts in this comment thread for now and eventually we can make a wiki page.

**Comment by sethaxen** [comment]
It would be really nice to get this established for PMI2.

**Comment by sethaxen** [comment]
The aforementioned developer guide can be found on the [wiki](https://github.com/salilab/pmi/wiki/Developer-Guide).
